x-pack/docs/en/security/authentication/built-in-users.asciidoc

```diff
@@ -91,6 +91,14 @@ new bootstrap password:
 bin/elasticsearch-keystore add "bootstrap.password"
 ----------------------------------------------------
 
+Alternatively you can set the bootstrap password by passing the password to stdin.
+This can be useful for scripted or automated deployments.
+
+[source,shell]
+------------------------------------------------------------------------------------------
+bash -c 'printf <PASSWORD> | bin/elasticsearch-keystore add bootstrap.password -x -f -v'
```

Comment:
Is there are reason you chose to explicitly pass this through `bash -c`? 
It's not typical of our docs to do so, but perhaps there's an explicit reason to want it here.

Reply:
I used bash -c as I had to run the command as sudo and if I just ran the command with 

'printf <PASSWORD> | bin/elasticsearch-keystore add bootstrap.password -x -f -v'

I would get a permissions issue. I believe that when running the command when passing it through bash -c allows the command after the pipe to also be run as sudo.